pkg/app/daemon/rebase.go

```diff
@@ -26,11 +26,11 @@ func (self *TodoLine) ToString() string {
 }
 
 func TodoLinesToString(todoLines []TodoLine) string {
-	lines := slices.Map(todoLines, func(todoLine TodoLine) string {
+	lines := lo.Map(todoLines, func(todoLine TodoLine, _ int) string {
```

Comment:
This mandatory index arg annoys the hell out of me but it's still worth it to use lo